lib/console/procps_utils.pm

```diff
@@ -39,7 +39,7 @@ sub read_memory_cpu {
             push @mem_num, $elem;
         }
     }
-    my @res_cpu = split '\n', script_output "awk '{print \$12}' $file_name";
+    my @res_cpu = split '\n', script_output "awk '{print \$13}' $file_name";
```

Comment:
which output you are checking here?

Reply:
us-  Percentage of CPU usage executing application code.  13th column in vmstat

Reply:
all your (previous than 17h ago) verification runs are done with the 12th column of the output?
If yes, you have to run them again with this new change :)

Reply:
Updated Latest run in the comment.